src/main/java/org/myrobotlab/service/Sphinx.java

```diff
@@ -1,700 +0,0 @@
-/**

```

Comment:
Please restore this file.  We shouldn't get rid of this class / service until we have another offline speech recognition alternative that is already borged in.

Reply:
it will be restored in another commit - i have no idea what the workyness of it is ...